test/api/dapps.js

```diff
@@ -848,7 +71,7 @@ describe('GET /dapps', function () {
 		});
 	});
 
-	it('using name should be ok', function (done) {
+	it.skip('using name should be ok', function (done) {
```

Comment:
ok I can see. Functionalities responsible for registering new dapps are removed so it is not possible to insert any of them before running the tests anymore. 

I think some another issue might be created for hardcoding them in DB, and then unskipipping these tests.